web/pgadmin/browser/server_groups/servers/databases/schemas/statistics/tests/test_statistics_add.py

```
@@ -0,0 +1,120 @@
+                    self.db_name,
+                    self.data["name"]
+                )
+
+                self.assertIsNotNone(
+                    cross_check_res,
+                    "Could not find the newly created statistics object."
+                )
+        else:
+            if self.mocking_required:
+                with patch(self.mock_data["function_name"],
+                           side_effect=eval(self.mock_data["return_value"])):
```

Ruff (0.15.6)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)